Adyen.Test/ManagementTest.cs

```diff
@@ -61,7 +61,7 @@ public async Task UpdateResource()
         public void ListTerminals()
         {
             var client = CreateMockTestClientApiKeyBasedRequestAsync("Mocks/management/list-terminals.json");
-            var service = new TerminalsTerminalLevelApi(client);
+            var service = new TerminalsTerminalLevelService(client);
```

Comment:
Love the -Service suffix 👍 

Note: Are you keeping track of these breaking changes btw? It would be nice to write a blog about it, how to migrate from a previous version to the current one. Example:
```
| Before                    | After                         |
| TerminalsTerminalLevelApi | TerminalsTerminalLevelService |
```